todo_app/templates/index.html

```diff
@@ -34,15 +34,23 @@ <h2>To-Do List</h2>
           </thead>
           <tbody>
             <tr>
-                    <td><li class="list-group-item">{{item.title}}</td>
-                    <td><li class="list-group-item">{{ item.status }}</td>
-                    <td><form id="toggle_item_{{ item.id }}" action="/toggle_status" method="POST">
+                    <td><li class="list-group-item">{{item.name}}</td> 
+                    <td><li class="list-group-item">{% for list in lists %}
+                      {% if item.idList == list.id %}
+                      {{list.name}}
+                      {% endif %}
+                    {% endfor %}
+                    <td><form id="toggle_item_{{ item.id }}" action="/in_progress" method="POST">
                       <input type="hidden" name="item_id" value="{{ item.id }}">
-                      <a onclick="document.getElementById('remove_item_{{ item.id }}').submit()"><button submit="remove_item()" class="btn btn-outline-success btn-block" style="margin-top:5px">Completed</button>
+                      <a onclick="document.getElementById('in_progress{{ item.id }}').submit()"><button submit="in_progress" class="btn btn-outline-primary btn-block" style="margin-top:5px">In Progress</button>
```

Comment:
Get rid of the `<a>` and the form id. It's unnecessary complication.

If all you wanted was for the button to submit the form,  make the button `<button type="submit"` or `<input type="submit" ...`. A button already submits the form (in some browsers at least) and you're in danger of making two POST requests if your onclick actually matched the form id.

You also haven't closed the `a` tag, though most browsers can work out what you meant.

Reply:
<a> and onclick have now been removed.